src/engine/config/config-handler.ts

```diff
@@ -70,6 +71,57 @@ export async function loadGameConfigurations(): Promise<void> {
 }
 
 
+/**
+ * find all items in all select groups
+ * @param groupKey string or array of string of which to find items connected with
+ * @return itemsKeys array of itemkeys in all select groups
+ */
+export const findItemTagsInGroup = (groupKey: string | string[]): string[] => {
```

Comment:
I'd use the plural version just so it's clear we're talking about an array of multiple entries. Same goes for all the rest in this file